tests/library/page-event-crash.spec.ts

```diff
@@ -70,9 +70,8 @@ test('should cancel navigation when page crashes', async ({ server, page, crash
   expect(error.message).toContain('page.goto: Page crashed');
 });
 
-test('should be able to close context when page crashes', async ({ isAndroid, isElectron, isWebView2, page, crash }) => {
+test('should be able to close context when page crashes', async ({ isAndroid, isWebView2, page, crash }) => {
   test.skip(isAndroid);
-  test.skip(isElectron);
```

Comment:
Can we keep these ones with the electron/browser version check instead? I find such historical annotations very useful later on.